src/test/mediapipeflow_test.cpp

```diff
@@ -1900,12 +1901,14 @@ TEST_F(MediapipeConfigChanges, ConfigWithEmptyBasePath) {
     std::string graphPbtxtFileContent = pbtxtContent;
     std::string configFileContent = configFileWithEmptyBasePath;
     std::string configFilePath = directoryPath + "/config.json";
-    std::string graphFilePath = directoryPath + "/graph.pbtxt";
+    std::string graphFilePath = directoryPath + "/mediapipeGraph/graph.pbtxt";
```

Comment:
```suggestion
    std::string graphName = "mediapipeGraph";
    std::string graphFilePath = directoryPath + "/" + graphName + "/" + MediapipeGraphConfig::DEFAULT_GRAPH_CONFIG_FILE_NAME;
```

Reply:
That would also require exposing static MediapipeGraphConfig class field. Right now it is just a locally scoped variable DEFAULT_GRAPH_FILENAME in mediapipegraphconfig.cpp.